pennylane/_qubit_device.py

```diff
@@ -66,6 +68,9 @@
 from pennylane.tape import QuantumScript, QuantumTape
 from pennylane.wires import Wires
 
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.NullHandler())
```

Comment:
The decision for that comes from here https://docs.python.org/3/howto/logging.html#configuring-logging-for-a-library

Basically, if a user decides to extend/depend on PennyLane internals, it can cause headaches with log-record reporting. The default library import behaviour is to define a null-handler object to throw away any log-records that are received --- the equivalent of piping things to `/dev/null`